docs/getting-started/getting-started-with-wdl.rst

```diff
@@ -208,11 +208,56 @@ the Dockstore command line and descriptor rather than just directly
 calling it via Docker. This will test that the WDL correctly describes
 how to run your tool.
 
+Setting Up the Dockstore CLI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
 The first thing I'll do is
 `setup the Dockstore CLI locally <https://dockstore.org/quick-start>`__.
 This will have me install all of the dependencies needed to run the
 Dockstore CLI on my local machine.
 
+Although not strictly required, it's a good idea to set a concurrent-job-limit when running locally. This is because Cromwell, when running locally, loses the ability to set compute resources within what is specified in a task's runtime attributes. As such, the local version of Cromwell sometimes uses too many resources when running on scattered tasks, causing the task to get sigkilled or for Docker to lock up. Of course, there is a tradeoff: If you set concurrent-job-limit, tests involving scattered tasks may execute slower as less instances of a scattered task will run in parallel.
+
+.. tip::  If a Docker lockup happens, you will notice tasks do not progress beyond WaitingForReturnCode and you will be temporarily unable to use Docker on your OS. Thankfully, this can be solved easily by restarting Docker on your machine.
+
+The easiest way to avoid these issues is to set up a Cromwell configuration file that sets concurrent-job-limit.
+
+1. Download `this template file <https://github.com/broadinstitute/cromwell/blob/develop/cromwell.example.backends/cromwell.examples.conf>`__ and name it ``.cromwell.conf``
```

Comment:
Is that a convention, to make it `.cromwell.conf`? If it's not a convention, I would be inclined to just call it `cromwell.conf`.

Reply:
Don't feel strongly, but could also consider making things a bit simpler by always picking a specific location like `.dockstore/cromwell.conf`

Reply:
My understanding was that the convention is .cromwell.conf because that's what the Cromwell jar will look for by default. In the context of the Dockstore CLI it's a moot point as we have to point to it directly anyway, but I think keeping the convention will help people who may switch runners. (That said, if we set up the Dockstore CLI to come with the template file itself, I do think `.dockstore/` is a good place for it.)

But now that I look again, it seems that maybe the jar would need to be passed the file anyway? https://cromwell.readthedocs.io/en/stable/tutorials/ConfigurationFiles/